client/src/pages/proposalDetails/ProposalDetails.js

```diff
@@ -53,15 +49,34 @@ function amountToDollarString(amount) {
 
 function ProposalConditionalRender(isAdmin) {
 
-  var VOTE_VALUE = false;
-  var PROGRESS_VALUE = !VOTE_VALUE;
-
-  const [showProgressBars, setToggleView] = useState(VOTE_VALUE);
+  var PROGRESS_VALUE = false;
+  const [showProgressBars, setToggleView] = useState(!PROGRESS_VALUE);
 
     /** Toggles view for admin conditional frame; returns progress bars
      * if admin toggle is set to "progress" and admin voting buttons if set to "vote" */
     function ToggleComponents(showProgress) {
+      const [AMOUNT_VOTED, setVotesTotal] = useState(0);
+      const [TOTAL_MEMBERS, setTotalMembers] = useState(1);
+      const [PERCENT_NO, setPercentNo] = useState(0);
+      const [PERCENT_YES, setPercentYes] = useState(0);
+      const [PERCENT_UNVOTED, setPercentUnvoted] = useState(0);
```

Comment:
General programming conventions are to reserve all caps + underline variable names for constant values (i.e. PI or YES_VOTE). Since these are state variables that will change, it's better to leave them in camel case and follow the state hook naming scheme as mentioned above ([hookName, setHookName])